packages/core/locales/zh-CN/zod.json

```diff
@@ -72,7 +73,8 @@
     "url": "url",
     "uuid": "uuid",
     "cuid": "cuid",
-    "regex": "正则表达式"
+    "regex": "正则表达式",
+    "datetime": "datetime"
```

Comment:
@izayoi-hibiki I would appreciate it if you could help me with the translation here and check line 22